blog/settings.py

```diff
@@ -164,9 +160,17 @@
 AUTH_USER_MODEL = 'account.User'
 
 EMAIL_USE_TLS = True
-EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+ENV = os.environ['ENV']
+if ENV.lower() == 'dev':
+    EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+else:
+    EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
 EMAIL_HOST = 'smtp.gmail.com'
-EMAIL_HOST_PASSWORD = 'geoadmin@123'
-EMAIL_HOST_USER = 'blog.vh@gmail.com'
+EMAIL_HOST_PASSWORD = 'qwerty6446'
+EMAIL_HOST_USER = 'asite249@gmail.com'
 EMAIL_PORT = 587
 DEFAULT_FROM_EMAIL = EMAIL_HOST_USER
+
+BASE_URL = os.environ["BASE_URL"]
+PASSWORD_RESET_TIMEOUT = 900
```

Comment:
You need another variable for profile activate timeout, and load them from env vars